client/app/bundles/App/GettingStarted/ThemingIntro.md

```diff
@@ -0,0 +1,5 @@
+## Theming
+
+Terra UI provides theming capabilities in various components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+
+### Themable Components
```

Comment:
was there more to this page?

Reply:
I'm using the header styles from the github.css file for this heading then mapping through terra-site/themable-variables.json and rendering those out below this. Here is how this looks in the browser:

<img width="1435" alt="screen shot 2017-09-07 at 2 56 15 pm" src="https://user-images.githubusercontent.com/633148/30182704-6254aa3a-93dd-11e7-9238-56b19b3041c4.png">

Reply:
Ahh looks good!